app/consumer-democracy/app.go

```diff
@@ -677,6 +677,7 @@ func New(
 				}
 			}
 
+			// TODO: remove this code
```

Comment:
TODO: is any of the code below needed?

Reply:
Not sure I understand. Why are we calling InitGenesis. This should be automatically called when adding a new module to the chain.